docs/monitoring-server.md

```diff
@@ -0,0 +1,141 @@
+<!--
+SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+
+SPDX-License-Identifier: Apache-2.0
```

Comment:
Documentation license should be CC-BY-SA-4.0

Reply:
Good catch. Actually it looks like all of the other documentation in this repo is also using apache license which should be changed @henrirosten

Reply:
True, we should change the documentation licenses. I propose we do this repo-wide with a separate PR after this has been merged.

Reply:
I noticed this earlier this week, there's already Jira ticket about it.